pkg/tests/tasks/traffic/circuit_breaking_test.go

```diff
@@ -75,12 +74,19 @@ func TestCircuitBreaking(t *testing.T) {
 			c503 := getNumberOfResponses(t, msg, `Code 503.*`)
 			successRate200 := 100 * c200 / reqCount
 			successRate503 := 100 * c503 / reqCount
-
-			if util.IsWithinPercentage(c200, reqCount, 0.6, tolerance) && util.IsWithinPercentage(c503, reqCount, 0.4, tolerance) {
-				t.LogSuccessf("%d%% of responses were 200 OK, and %d%% were 503 Service Unavailable", successRate200, successRate503)
-			} else {
-				t.Fatalf("%d%% of responses were 200 OK, and %d%% were 503 Service Unavailable", successRate200, successRate503)
+			t.Log(fmt.Sprintf("Success rate 200: %d%%", successRate200))
+			t.Log(fmt.Sprintf("Success rate 503: %d%%", successRate503))
+			if successRate503 == 0 {
```

Comment:
Is 0 ok? What if the 503 is a result of something else? 

If we expect 30% of requests to return 503, we should probably require successRate503 to be > 10%. 

Btw, why is it called "successRate503", if the request is *not* successful.

Reply:
Ok, I'll change to validate that at least 5 request (10%) fails